web/src/components/zonemap.jsx

```diff
@@ -81,46 +79,24 @@ const ZoneMap = ({
     // We here iterate over the zones list (instead of dict) to keep the iteration
     // order stable
     const features = zoneValues.map((zone, i) => {
-      if (isAggregatedViewFFEnabled) {
-        if (isAggregateEnabled && zone.geography.properties.isAggregatedView) {
-          const length = (coordinate) => (coordinate ? coordinate.length : 0);
-          const zoneId = zone.config.countryCode;
-          return {
-            type: 'Feature',
-            geometry: {
-              ...zone.geography.geometry,
-              coordinates: zone.geography.geometry.coordinates.filter(length), // Remove empty geometries
-            },
-            id: i, // assign an integer id so the feature can be updated later on
-            properties: {
-              color: undefined,
-              zoneData: zone[selectedTimeAggregate].overviews,
-              zoneId,
-            },
-          };
-        }
-        if (!isAggregateEnabled && !zone.geography.properties.isCombined) {
-          const length = (coordinate) => (coordinate ? coordinate.length : 0);
-          const zoneId = zone.config.countryCode;
-          return {
-            type: 'Feature',
-            geometry: {
-              ...zone.geography.geometry,
-              coordinates: zone.geography.geometry.coordinates.filter(length), // Remove empty geometries
-            },
-            id: i, // assign an integer id so the feature can be updated later on
-            properties: {
-              color: undefined,
-              zoneData: zone[selectedTimeAggregate].overviews,
-              zoneId,
-            },
-          };
-        }
-        if (!zone.geography.properties.aggregatedView) {
-          return {};
-        }
+      if (isAggregateEnabled && zone.geography.properties.isAggregatedView) {
+        const length = (coordinate) => (coordinate ? coordinate.length : 0);
+        const zoneId = zone.config.countryCode;
+        return {
+          type: 'Feature',
+          geometry: {
+            ...zone.geography.geometry,
+            coordinates: zone.geography.geometry.coordinates.filter(length), // Remove empty geometries
+          },
+          id: i, // assign an integer id so the feature can be updated later on
+          properties: {
+            color: undefined,
+            zoneData: zone[selectedTimeAggregate].overviews,
+            zoneId,
+          },
+        };
       }
-      if (!isAggregatedViewFFEnabled) {
+      if (!isAggregateEnabled && !zone.geography.properties.isCombined) {
```

Comment:
Can we put a comment on this functionality perhaps? I am not sure I understand

Reply:
Done :D